sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/README.md

```diff
@@ -334,6 +334,33 @@ In scenarios involving multiple scopes or a single scope with multiple key-value
 only the first occurrence of the key-value pair from the outermost scope will be recorded.
 However, when the same key is utilized both within a logging scope and directly in the log statement, the value specified in the log message template will take precedence.
 
+### CustomEvents
+
+Azure Monitor relies on OpenTelemetry's Log Signal to create CustomEvents.
+For .NET, users will use ILogger and place an attribute named `"microsoft.custom_event.name"` in the message template.
+Severity and CategoryName are not recorded in the CustomEvent.
+
+#### via ILogger.Log methods
+
+To send a CustomEvent via ILogger, include the `"microsoft.custom_event.name"` attribute in the message template.
+
+Note: This example shows `LogInformation`, but any Log method can be used.
+Severity is not recorded, but depending on your configuration it may be filtered out.
+Users should take care to select a severity for CustomEvents that is not filtered out by their configuration.
+
+```csharp
+var loggerFactory = LoggerFactory.Create(builder =>
```

Comment:
This is not valid in distro's case. Registration of UseAzureMonitorExporter should help.

Reply:
You could get reference for your code from this demo - https://github.com/Azure/azure-sdk-for-net/blob/349ed73ad5079fc70cd0e6daed7f1ef609d43266/sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/tests/Azure.Monitor.OpenTelemetry.AspNetCore.Demo/Program.cs#L14

Reply:
fixed, thank you.